app/assets/stylesheets/03-navbar.scss

```diff
@@ -18,7 +18,7 @@
     left: 0px;
     padding-left: 60px;
     padding-right: 130px;
-    background-color: black;
+    background-color: #121212;
```

Comment:
might be worth putting this [into a variable](https://sass-lang.com/documentation/variables/) so you can easily change it later if needed

Reply:
Great point; thanks for mentioning. I'll have to look into SASS or CSS variables.